mevislab.github.io/content/contact.md

```diff
@@ -1,17 +1,19 @@
 ---
-title: "Contact Us"
+title: "Contact"
 date: 2022-06-15T08:54:53+02:00
 draft: false
 ---
-# Contact Us
-## General Questions
-For general questions regarding MeVisLab and especially for feedback on this documentation, please contact [info(at)mevislab.de](mailto://info@mevislab.de).
+### Feedback is valuable and always appreciated! 
 
-## Licensing
-For sales information regarding licenses, please contact [sales(at)mevislab.de](mailto://sales@mevislab.de).
+#### MeVisLab Licensing
+Having any questions on MeVisLab Licensing? Please contact the [MeVisLab Sales-Team](mailto://sales@mevislab.de)
 
-## Online Forum
-Additional support is available in the [MeVisLab Forum](https://forum.mevislab.de).
+#### MeVisLab Forum
+Searching for a Forum to ask your specific MeVisLab questions? Having trouble with functionalities? Ask [here](https://forum.mevislab.de)! Someone else might know the answer. If not - one of our developers will help you out!
+
+#### General Questions
+General questions regarding MeVisLab? Don´t hesitate to contact the [MeVisLab Team](mailto://info@mevislab.de).
```

Comment:
Please elaborate furtherly. I can't wrap my head around the structure of that sentence using "instead of". 
How about "Either one of our developers or another (fellow) MeVisLab user will help you out!"?

Reply:
Sorry - please use the straight apostroph `'` instead of the accent aigu `´` (whatever this is called in English).